app/models/comment.rb

```diff
@@ -1,4 +1,9 @@
 class Comment < ApplicationRecord
   belongs_to :user
   belongs_to :post
+
+  def update_comments_counter
+    post.comments_counter = post.comments.count
+    post.save
+  end
```

Comment:
- [ Optional ] Good Job for using [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_comments_counter` method every time a new comment has been saved to database, then **Kindly I recommend you to make the `update_comments_counter` method private**.